systemtest/rum_test.go

```diff
@@ -65,7 +66,23 @@ func TestRUMXForwardedFor(t *testing.T) {
 		"@timestamp", "timestamp.us",
 		// RUM events have the source port recorded, and in the tests it will be dynamic
 		"source.port",
+		// Do not assert the exact contents of the geo field since they can change slightly
+		// depending on the IP lookup.
+		"client.geo",
 	)
+
+	var geo = make(map[string]interface{})
+	var ip string
+	for _, doc := range result.Hits.Hits {
+		client := doc.Source["client"].(map[string]interface{})
+		ip = client["ip"].(string)
+		geo = client["geo"].(map[string]interface{})
+	}
+	assert.NotZero(t, geo)
+	assert.Contains(t, geo, "location")
+	assert.Contains(t, geo, "region_iso_code")
+	assert.Contains(t, geo, "country_iso_code")
```

Comment:
I chose to assert these fields being set since I feel like they won't likely change the format, but can also just remove the assertion.